### PR TITLE
Topic auto kafka

### PR DESCRIPTION
**Pull request by urviljoshi**

### CODE REVIEW

File: hub/hub_service.bal [hunk 1]
```diff
@@ -284,8 +284,7 @@ websubhub:Service hubService = service object {
         if !topicAvailable {
             log:printInfo("Topic not found - Auto registering topic", topic = topicName);
             websubhub:TopicRegistration topicRegistrationMsg = {
-                topic: topicName,
-                hubMode: "register"
+                topic: topicName
```
Comment (vishwa-vyom) [inline hub/hub_service.bal:287]: why are we removing hub mode ?

File: kafka-admin-client/src/test/java/com/example/kafkaadminclient/AppTest.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
+     */
+    public void testApp()
+    {
+        assertTrue( true );
```
Comment (vishwa-vyom) [inline kafka-admin-client/src/test/java/com/example/kafkaadminclient/AppTest.java:36]: What test case is this ? If there is no test case for now, let not add this file

File: kafka-admin-client/pom.xml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<maven.compiler.source>11</maven.compiler.source>
+		<maven.compiler.target>11</maven.compiler.target>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.apache.kafka</groupId>
+			<artifactId>kafka-clients</artifactId>
+			<version>2.8.0</version>
```
Comment (vishwa-vyom) [inline kafka-admin-client/pom.xml:24]: Is this the same version kafka client used by them ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: hub/hub_service.bal (hunk 1 above)
Comment (urviljoshi) [reply hub/hub_service.bal:287]: not needed in current version

File: kafka-admin-client/src/test/java/com/example/kafkaadminclient/AppTest.java (hunk 2 above)
Comment (urviljoshi) [reply kafka-admin-client/src/test/java/com/example/kafkaadminclient/AppTest.java:36]: removing

File: kafka-admin-client/pom.xml (hunk 3 above)
Comment (urviljoshi) [reply kafka-admin-client/pom.xml:24]: same

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**